lib/cli/commands-schema/common-options/global.js

```diff
@@ -3,6 +3,8 @@
 module.exports = {
   help: { usage: 'Show this message', shortcut: 'h', type: 'boolean' },
   version: { usage: 'Show version info', type: 'boolean' },
+  verbose: { usage: 'Show verbose logs', type: 'boolean' },
+  debug: { usage: 'Namespace of debug logs to expose (use "*" to display all)', type: 'string' },
```

Comment:
Will `--debug` work (without value)? Because `--debug=xxx` wasn't the plan

Reply:
We cannot reliably validate flag that goes with and without value, and with debug logs in most cases you'd want to filter against some namespace.

Since beginning I was communicating that debug logging is namespace based feature

Reply:
OK, please update the initiative to match that change then.

Reply:
The initiative just mentions those flags, but doesn't specify them (there's no word on what types they are). So technically it's not against the initiative. Anyway I've added details on that there

Reply:
> The initiative just mentions those flags, but doesn't specify them (there's no word on what types they are).

That's because it wasn't supposed to include a value initially (i.e. they were simple boolean flags).

Thanks for updating.